README.md

```diff
@@ -88,6 +88,16 @@ Possible values: `true`, `false`
 
 Compress the resulting GeoJSON or TopoJSON with Geobuf.
 
+#### options.rename
```

Comment:
There is no default value, if the function is not set then it will not be run. It would then just use the normal default value in the files option